tests/mypy_test.py

```diff
@@ -76,6 +76,13 @@ def valid_path(cmd_arg: str) -> Path:
     return path
 
 
+def remove_dev_suffix(version: str) -> str:
+    """Helper function for argument-parsing"""
+    if version.endswith("-dev"):
```

Comment:
`.removesuffix`? Or do we run this on overly old versions?

Reply:
We run this on py37, yeah :)

I can add a comment if you like? The reason why we run this on py37 is because if we run mypy with the `--python-version=3.7` argument _on py311_ (or whatever), pip will install a version of numpy that's incompatible with python 3.7 (because the latest version of numpy is incompatible with 3.7). This then causes mypy to complain about invalid syntax on some of the types it finds in `site-packages/` when it's run with the `--python-version=3.7` flag.

Reply:
Thanks! Let's just leave this as is then.